common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -934,11 +933,12 @@ div.problem .action {
   }
 
   .submit-attempt-container {
-    display: table;
+    display: inline-block;
 
     @media (max-width: $bp-screen-lg) {
       max-width: 100%;
       padding-top: $baseline;
```

Comment:
Can you try removing the padding-top now? It was probably just there for the buttons wrapping above it, and I think it may be adding to extra spacing between the actual problem text and the buttons.